client/python/redact_fields.yml

```diff
@@ -56,6 +56,8 @@
         - namespace
         - name
         - field
+    - class_name: Transformation
+      redact_fields: []
```

Comment:
I guess we should add all the fields to skip them on the redaction (maybe `description` could contain secrets in some weird case?) WDYT @mobuchowski?

Reply:
Yeah I agree. I don't see how this could somehow contain any sensitive information, unless explicitely embedded in SQL code.